### PR TITLE
Fixed GMMK Pro -> stickandgum keymap readme.md

### PR DESCRIPTION
**Pull request by stickandgum**
Very minor edits to readme.md mostly to reflect NKRO no longer default.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [x] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [ ] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [ ] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [ ] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/gmmk/pro/ansi/keymaps/stickandgum/config.h [hunk 1]
```diff
@@ -0,0 +1,75 @@
+/* Copyright 2021 stickandgum
```
Comment (fauxpark) [inline keyboards/gmmk/pro/ansi/keymaps/stickandgum/config.h:1]: Needs to use Unix line endings.

File: keyboards/gmmk/pro/ansi/keymaps/stickandgum/readme.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
+# Glorious GMMK Pro / ANSI - Enhanced Keyboard
```
Comment (fauxpark) [inline keyboards/gmmk/pro/ansi/keymaps/stickandgum/readme.md:1]: Needs to use Unix line endings.

File: keyboards/gmmk/pro/ansi/keymaps/stickandgum/rules.mk [hunk 3]
```diff
@@ -0,0 +1,2 @@
+CONSOLE_ENABLE = no
```
Comment (fauxpark) [inline keyboards/gmmk/pro/ansi/keymaps/stickandgum/rules.mk:1]: Needs to use Unix line endings.

**Review by fauxpark — CHANGES_REQUESTED** [review]

**Review by noroadsleft — APPROVED** [review]

**Review by fauxpark — APPROVED** [review]

**Comment by drashna** [comment]
Thanks!

**Pull request merged**